packages/webpack/test/scaffold.js

```diff
@@ -120,6 +128,7 @@ function runScript(code, globals = defaultGlobals()) {
     throw new Error('runScript requires a bundle string as the first argument')
   }
   const context = createContext(globals)
+  context.self = context // minimal browser emulation
```

Comment:
what does this mean?

Reply:
browsers have a `self` global that some things rely on.